app/styles/partials/nav-bar.scss

```diff
@@ -23,6 +23,11 @@
   left: auto !important;
   right: 0 !important;
 
+  .icon {
```

Comment:
`icon` is a semantic ui class. Please use some other class as this will apply to all the elements using `icon` class.

Reply:
Fixed.